ui/apps/platform/src/reducers/telemetryConfig.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-underscore-dangle */
+import { combineReducers } from 'redux';
+
+import { createFetchingActionTypes } from 'utils/fetchingReduxRoutines';
+import { fetchTelemetryConfig } from 'services/TelemetryConfigService';
+import { analyticsIdentity } from 'utils/analytics';
+
+import { initializeSegment } from 'global/initializeAnalytics';
+
+// Types
+
+export const types = {
+    FETCH_TELEMETRY_CONFIG: createFetchingActionTypes('telemetryConfig/FETCH_TELEMETRY_CONFIG'),
+};
+
+// Actions
+
+export const fetchTelemetryConfigThunk = () => {
+    return async (dispatch) => {
+        dispatch({ type: types.FETCH_TELEMETRY_CONFIG.REQUEST });
+
+        try {
+            const result = await fetchTelemetryConfig();
+            initializeSegment(result.response.storageKeyV1);
+            analyticsIdentity(result.response.userId);
+
+            dispatch({
+                type: types.FETCH_TELEMETRY_CONFIG.SUCCESS,
+                response: result.response,
+            });
+        } catch (error) {
+            dispatch({ type: types.FETCH_TELEMETRY_CONFIG.FAILURE, payload: error });
+        }
+    };
+};
+
+// Reducers
+
+const telemetryConfig = (state = [], action) => {
+    if (action.type === types.FETCH_TELEMETRY_CONFIG.SUCCESS) {
+        return action.response.telemetryConfig ?? state;
+    }
+    return state;
+};
+
+const error = (state = null, action) => {
+    switch (action.type) {
+        case types.FETCH_TELEMETRY_CONFIG.REQUEST:
+        case types.FETCH_TELEMETRY_CONFIG.SUCCESS:
+            return null;
+
+        case types.FETCH_TELEMETRY_CONFIG.FAILURE:
+            return action.error;
```

Comment:
If I'm not mistaken, it looks like the property that contains the error on the action is `payload` based on line 32 above.

Reply:
You're right. I did the same thing in another PR.. Updated